app/helpers/application_helper.rb

```diff
@@ -2,4 +2,18 @@ module ApplicationHelper
   def format_date_time(cert_date_time)
     cert_date_time.strftime("%e-%m-%Y %H:%M")
   end
+
+  def hub_environments
+    JSON.parse(ENV.fetch('HUB_ENVIRONMENTS'))
+  rescue KeyError
+    Rails.logger.error('No hub environment buckets found')
```

Comment:
what about sticking this to `/config/environments/*` as values instead of the function ? or is there an use case when we want to configure it locally?